recipe/conda_build_config.yaml

```diff
@@ -330,7 +330,8 @@ pin_run_as_build:
   xz:
     max_pin: x.x
   zeromq:
-    max_pin: x.x
+    max_pin: x.x  # [not win]
+    max_pin: x.x.x  # [win]
```

Comment:
Are these in `run_exports` for all builds of `4.3.2` ?

Reply:
Only in build 2. Probably best to mark builds 0 and 1 (all platforms) as broken, since there was a typo in the run_exports preventing them from being correct.

Reply:
> Only in build 2. Probably best to mark builds 0 and 1 (all platforms) as broken, since there was a typo in the run_exports preventing them from being correct.

Done. See https://anaconda.org/conda-forge/zeromq/files?version=4.3.2